src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -10,6 +10,14 @@
     @Config.Name("Unofficial Options")
     public static UnofficialOptions U = new UnofficialOptions();
 
+    @Config.Comment("Whether to enable that Steam Multiblocks use Steel instead of Bronze. Default: false")
+    @Config.RequiresMcRestart
+    public static boolean steelMultiblocks = false;
```

Comment:
Please change this config name to "steelSteamMultiblocks". It's more clear on its purpose.